app/routes/pages/components/LandingPage.tsx

```diff
@@ -14,7 +15,9 @@ import TextWithBoldTitle, {
 } from './subcomponents/TextWithTitle';
 import type { PageRenderer } from 'app/routes/pages/components/PageDetail';
 
-const LandingPage: PageRenderer = ({ loggedIn }) => {
+const LandingPage: PageRenderer = () => {
+  const { loggedIn } = useUserContext();
```

Comment:
This actually resolves a kinda important bug. `loggedIn` would always be `undefined`, so the info and link to our "varslingsportal" would never be shown to any users ..